Tests/FeatherWidgets.TestIntegration/Navigation/LanguageSelectorWidgetTests.cs

```diff
@@ -157,6 +329,12 @@ private string GetPageUrl(string pageName, CultureInfo culture, bool isDefaultCu
             }
         }
 
+        private string GetPageUrlOfNotTranslatedPages(string homePageName, CultureInfo culture, CultureInfo homePageCulture)
```

Comment:
When a page does not have a translation, the language selector widget will create a link that will look like: /{wanted culture}/{the page name in the default culture}. Passing a homePageCulture is confusing, because the home page can have multiple cultures. You do not need culture, you need the name of the default culture page. So the method's parameters can be defaultCultureHomePageName and culture.